web/src/main/java/org/mskcc/cbio/oncokb/api/pvt/PrivateSearchApiController.java

```diff
@@ -368,7 +368,14 @@ private static TypeaheadSearchResp newTypeaheadCancer(CancerTypeMatch cancerMatc
                 typeaheadSearchResp.setHighestResistanceLevel(highestLevel.getLevel());
             }
 
-            typeaheadSearchResp.setLink("actionableGenes#sections=Tx&tumorType=" + cancerMatch.getCancerType().getCode());
+            String cancerType = cancerMatch.getCancerType().getCode();
+            if (StringUtils.isEmpty(cancerType)) {
+                cancerType = cancerMatch.getCancerType().getSubtype();
+            }
+            if (StringUtils.isEmpty(cancerType)) {
+                cancerType = cancerMatch.getCancerType().getMainType();
+            }
+            typeaheadSearchResp.setLink("actionable-genes#sections=Tx&cancerType=" + cancerType);
```

Comment:
The parameter name change is related to https://github.com/oncokb/oncokb-public/pull/1030